src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -3322,15 +3322,15 @@ def create_or_update_tag_at_scope(cmd, resource_id=None, tags=None, tag_name=Non
         tag_obj = Tags(tags=tags)
         TagsResource = cmd.get_models('TagsResource')
         tags_resource = TagsResource(properties=tag_obj)
-        return rcf.tags.create_or_update_at_scope(scope=resource_id, parameters=tags_resource)
+        return rcf.tags.begin_create_or_update_at_scope(scope=resource_id, parameters=tags_resource)
```

Comment:
Shouldn't adding `begin_` prefix be done during Track 2 migration? Does this `create_or_update_at_scope` work currently?

Reply:
This should be due to the sudden breaking change in the Python SDK, which forced us to do the compatibility logic. Before that, the resource module had completed the migration of most sub RPs.
The method `create_or_update_at_scope` no longer exists in the Python SDK, it is directly replaced by `begin_create_or_update_at_scope`.
@Jing-song Could you please help confirm whether my explanation is correct?